### PR TITLE
general.yml: change default contact nickname and mail to info@berlin.*

### PR DESCRIPTION
**Pull request by booo**
As discussed earlier I changed the standard contact information.

### CODE REVIEW

**Comment by Akira25** [comment]
I'd rather prefer to have the direct contact to the location operator there. This was the reason for the last change in a6cee5c9c9fabea7b4118054da755f015054b402. If the operator doesn't want to add its email publicly, he/she can add the email-contact-formular link from the config wizard.

In the mentioned commit you can find some template-code too, which would get broken by this commit.

File: group_vars/all/general.yml [hunk 1]
```diff
@@ -35,4 +35,4 @@ sysupgrade_preserve_custom_files:
   - /etc/wireguard/wg.key
   - /root/
 
-image_search_pattern: "*-sysupgrade.*"
\ No newline at end of file
+image_search_pattern: "*-sysupgrade.*"
```
Comment (Akira25) [inline group_vars/all/general.yml:38]: What is this line for? Did it maybe accidentially get into this commmit?

**Comment by PolynomialDivision** [comment]
@Akira25 We talk here about default values. It is just the "fallback" and not the default. Users should specify contact information in their location. :)

File: group_vars/all/general.yml (hunk 1 above)
Comment (PolynomialDivision) [reply group_vars/all/general.yml:38]: Look here:
https://github.com/Freifunk-Spalter/bbb-configs/commit/49b41b4b4813315bb7cfc9587cd3ffe551f53f31

If you see the diff, philipps editor or whatever just added the newline at the end.

**Comment by Akira25** [comment]
> @Akira25 We talk here about default values. It is just the "fallback" and not the default. Users should specify contact information in their location. :)

I know that. But in the past it hast shown, that most users, especially @freifunkoperator, rely on those default-values. To set an incentive to set the right maintainer contact, We've made the last change to this file, which was approved by @spolack.

To get the people setting right values, the standard value shouldn't make sense.

**Comment by PolynomialDivision** [comment]
Locations with contact information:
- k9
- kiehlufer
- fsev
- gubener
- huette

My opinion is, if all our ssh keys are on it, the email should go to all of us. I would prefer that and it helps if a maintainer becomes unresponsive. Lets share work and knowledge. ;)

**Comment by Akira25** [comment]
> Locations with contact information:
> 
> * k9
> * kiehlufer
> * fsev
> * gubener
> * huette
> 
in short: every location, that doesn't get maintained by you. ;-)

> My opinion is, if all our ssh keys are on it, the email should go to all of us. I would prefer that and it helps if a maintainer becomes unresponsive. Lets share work and knowledge. ;)

Maybe we can add the info-address as second e-mail-address. Ever since the bbb-net started, there was for every location a maintainer. If there is a person, that mainly maintains a location, we should mention this! If that is subject to change, it's really easy, to change that. There is no need to make maintainers "unavailable" by hiding them behind a general e-mail address, which they even don't get the mails from.

**Comment by booo** [comment]
The bbb installations are mostly community "owned" and maintained. There are specific maintainers or people that feel extra responsible for an installation and they can replace the contact if they like. As default a general mail address is good because we maintain the installations as a team and the team/community should be reachable by this mail.

We can add our mailinglist too or add a paragraph to the other fields in the contact site. Maybe a link to the wiki would be nice too.

**Comment by booo** [comment]
Maybe we should discuss this in a meeting and find some consensus for a minimal standard.

**Comment by spolack** [comment]
I'd like to have the mailing list as default. Its better to have the list for community-owned locatinos instead of dead contacts from inactive persons.

For all "@home" locations we should enforce setting a contact that by using review process.

**Comment by booo** [comment]
Problem with the mailinglist is that you have to register first to be able to write to the mailinglist.

By the way we can update the info@ list at some point.

**Comment by Akira25** [comment]
>For all "@home" locations we should enforce setting a contact that by using review process.

Lets do it that way. I've adjusted the template accordingly.

**Pull request merged**